src/sass/components/_section-feedback.scss

```diff
@@ -0,0 +1,47 @@
+.section-feedback {
+  .container {
+    .panel-bar {
```

Comment:
Czy potrzebujemy tę pustą regułkę?

Reply:
Jak tworzyłem początkowy schemat zapomniałem później usunąć. Już to poprawiłem